samples/RazorSlices.Samples.WebApp/Program.cs

```diff
@@ -85,6 +86,37 @@
         : TypedResults.NotFound();
 });
 
+
+var htmxTodos = app.MapGroup("/htmx-todo");
+var renderTodoTable = () => Results.Extensions.RazorSlice<Slices.HtmxTodos._Todos, List<Models.Todo>>(Models.Todos.AllTodos);
+htmxTodos.MapGet("/", () => Results.Extensions.RazorSlice<Slices.HtmxTodos.TodoIndex, List<Models.Todo>>(Models.Todos.AllTodos));
+htmxTodos.MapPost("/", ([FromForm] string title) => {
+    var maxId = Models.Todos.AllTodos.Select(x => x.Id).DefaultIfEmpty(0).Max();
+    Models.Todos.AllTodos.Add(new Models.Todo { Id = maxId + 1, Title = title });
+    return renderTodoTable();
+}).DisableAntiforgery();
```

Comment:
Don't like disabling antiforgery. Can you update the HTMX slice to inject the `IAntiforgery` service and emit the token?